src/coreclr/nativeaot/Runtime/stressLog.cpp

```diff
@@ -80,10 +81,125 @@ const static unsigned __int64 RECYCLE_AGE = 0x40000000L;        // after a billi
 
 /*********************************************************************************/
 
+#ifdef MEMORY_MAPPED_STRESSLOG
```

Comment:
What would it take to refactor the stresslog implementation to get rid of this duplicated copy and have one implementation that is shared by all runtimes?

Reply:
It is indeed a good win from an engineering perspective. I envision if we had a libStresslog, it could be repurposed for other projects as well. 

I did consider that option when I started, but I decided that it was too costly for the quality week. From an implementation perspective, I imagine it will follow a similar path to what we did with eventpipe.

Reply:
I would rather we take the time to do this right than introducing more duplicated code.